Package.swift

```diff
@@ -17,24 +17,13 @@ let package = Package(
             name: "SendGridKit",
             dependencies: [
                 .product(name: "AsyncHTTPClient", package: "async-http-client"),
-            ],
-            swiftSettings: swiftSettings
+            ]
         ),
         .testTarget(
             name: "SendGridKitTests",
             dependencies: [
                 .target(name: "SendGridKit"),
-            ],
-            swiftSettings: swiftSettings
+            ]
         ),
     ]
 )
-
-var swiftSettings: [SwiftSetting] { [
-    .enableUpcomingFeature("ExistentialAny"),
```

Comment:
FYI `ExistentialAny` is not included in Swift 6, so keeping it can have some value.

Reply:
This.

Reply:
Ok, I'm putting it back in 👍🏻